apps/e2e/lookup-storybook/src/app/lookup/lookup.component.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { LookupComponent } from './lookup.component';
+import { LookupModule } from './lookup.module';
+
+describe('LookupComponent', () => {
```

Comment:
If you pull in `main` this file can now be removed as we are no longer running the test target.